shaptools/hana.py

```diff
@@ -448,3 +449,106 @@ def get_sr_status(self):
         # (see SAPHana RA)
         status["status"] = SR_STATUS.get(result.returncode, SR_STATUS[12])
         return status
+
+    def set_ini_parameter(
+            self, ini_parameter_values, database, file_name, layer,
+            layer_name=None, reconfig=False,
+            key_name=None, user_name=None, user_password=None):
+        """
+        Set HANA configuration parameters in ini file
+
+        SQL syntax:
+        ALTER SYSTEM ALTER CONFIGURATION (<filename>, <layer>[, <layer_name>])
+        SET (<section_name_1>,<parameter_name_1>) = <parameter_value_1>,
+            (<section_name_2>,<parameter_name_2>) = <parameter_value_2>
+        WITH RECONFIGURE
+
+        key_name or user_name/user_password parameters must be used
+        Args:
+            ini_parameter_values(list): list containing HANA parameter details
```

Comment:
I don't like much this being and list of lists, it makes a bit difficult to understand.
What I would do instead is, change the list elements by dictionaries like:
```
{'section_name':'name', 'parameter_name':'param_name', 'parameter_value':'value'}
```
This way actually, you could have the same set and unset methods coded the same. Depending on the set/unset value, you would use only some entries in the dictionary or all of them (you could do that with the current solution anyway)

In fact, you could add a small sanity check to the incoming parameters to check correctness